msteams-platform/m365-apps/actions-in-m365.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Actions in Microsoft 365
+description: In this article, learn more about the function of Actions and its use cases. 
+ms.date: 11/16/2023
+ms.author: mosdevdocs
+author: Vishnu
+ms.topic: Conceptual
+ms.subservice: m365apps
+---
+# Actions in Microsoft 365
+
+> [!NOTE]
+>
+> * Actions for Microsoft 365 is available in [public developer preview](../resources/dev-preview/developer-preview-intro.md).
+>
+> * Actions is supported for Microsoft 365 apps on web and desktop clients only and isn't supported on Outlook and Microsoft Teams.
```

Comment:
Actions preview is supported in the Microsoft 365 (Office) app for [web ](http://www.microsoft365.com/) and [desktop](https://www.microsoft.com/store/productId/9WZDNCRD29V9).

(No need to enumerate all the hosts it's not currently supported)